index.js

```diff
@@ -2,8 +2,12 @@ module.exports = function(precompile) {
   return function(babel) {
     var t = babel.types;
 
-    var replaceNodeWithPrecompiledTemplate = function(node, template) {
-      var compiledTemplateString = "Ember.HTMLBars.template(" + precompile(template) + ")";
+    var replaceNodeWithPrecompiledTemplate = function(node, template, file) {
+      var moduleName = file.opts && file.opts.filenameRelative;
+
+      var compiledTemplateString = "Ember.HTMLBars.template(" + precompile(template, {
+        moduleName: moduleName
+      }) + ")";
```

Comment:
So `moduleName` will be `unkown`.. Do you know if and how this will interfere with custom transformations which rely on this option?